run.py

```diff
@@ -4,5 +4,5 @@
 
 
 if __name__ == '__main__':
-    app.run()
+    app.run(host='0.0.0.0)
```

Comment:
Hey @davidmukiibi, I think you forgot to add a closing quote at the end of the last zero and also do you mind explaining why we are changing this to `host='0.0.0.0'`?

Reply:
The app running that way makes it deployable to the cloud and other tools such as docker or docker compose.

Reply:
@chadwalt